src/components/artist.tsx

```diff
@@ -4,98 +4,109 @@ import Link from "next/link";
               target="_black"
               rel="noopener noreferrer"
+              role="button"
             >
               <Image
                 className={
-                  "mx-4 rounded-xl bg-gradient-to-bl from-fuchsia-300 to-amber-200 p-1 hover:from-fuchsia-500 hover:to-amber-400 active:from-fuchsia-700 active:to-amber-600 md:mx-12"
+                  "rounded-xl p-1 hover:bg-green-400 active:bg-green-600"
                 }
-                src={`/buttons/instagram_icon.png`}
-                alt="Link do IG artysty."
-                width="64"
-                height="64"
+                src={`/buttons/spotify-svgrepo-com.svg`}
+                alt="Link do Spotify artysty."
+                width="43"
+                height="43"
               />
             </Link>
-          ) : null}
-          {/* spotify button */}
-          {spotifyUrl ? (
+          )}
+          {/* instagram button */}
+          {Boolean(instagramUrl) && (
             <Link
-              href={spotifyUrl}
+              role="button"
+              href={instagramUrl}
               target="_black"
               rel="noopener noreferrer"
-              role="button"
             >
               <Image
                 className={
-                  "mx-4 rounded-xl bg-green-200 p-1 hover:bg-green-400 active:bg-green-600 md:mx-8"
+                  "rounded-xl bg-gradient-to-bl p-1 hover:from-fuchsia-500 hover:to-amber-400 active:from-fuchsia-700 active:to-amber-600"
                 }
-                src={`/buttons/spotify_logo_full.png`}
-                alt="Link do Spotify artysty."
-                width="192"
-                height="96"
+                src={`/buttons/ig.svg`}
+                alt="Link do Instagrama artysty."
+                width="43"
+                height="43"
               />
             </Link>
-          ) : null}
+          )}
         </div>
-        {/* event summary for the artist */}
-        <div className="m-2 rounded-xl bg-gray-200 p-1">
-          <h1 className="font-semibold">Kiedy i gdzie:</h1>
-          <div className="flex flex-col items-center">
-            {events.map((event) => (
-              <span key={event.id}>
-                Dzień {event.events_id.day} |{" "}
-                {event.events_id.start_time.slice(0, 5)} |{" "}
-                {event.events_id.location.name}
-              </span>
-            ))}
-          </div>
-
-          {/* zobacz w harmonogramie, encapsulated in div because margin did not work for Link */}
-          <div className="m-2">
-            <Link
-              href={`/schedule`}
-              className="rounded-lg bg-stone-700 p-2 font-semibold text-white hover:bg-stone-950"
+      </div>
+      <div className="w-full pr-4 text-sm lg:text-xl">
+        {events.length > 0 ? (
+          events.slice(0, 1).map((event) => (
+            <div
+              key={event.id}
+              className="mx-2 flex w-full justify-between text-sm lg:text-xl"
```

Comment:
Dodalabym tutaj jeszcze jeden breakpoint 
```suggestion
              className="mx-2 flex w-full justify-between text-sm lg:text-l xl:text-xl"
```